src/Cli/dotnet/commands/dotnet-workload/InstallStateContents.cs

```diff
@@ -12,17 +14,21 @@ internal class InstallStateContents
         public bool? UseWorkloadSets { get; set; }
 
         [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
-        public Dictionary<string, string> Manifests { get; set; }
+        public Dictionary<string, string>? Manifests { get; set; }
+
+        [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
+        public string? WorkloadSetVersion { get; set; }
```

Comment:
I think I mentioned this when we were discussing the PR, but I may have forgotten to include it in the PR comments: The property name in the global.json file should be `workloadVersion` (though it looks like the tests weren't consistent with the casing, so probably we should be case-insensitive).

So this property should be named `WorkloadVersion`, or I think you could also use a Json attribute to change the name used when reading / writing json.